src/styles/Specifications.css

```diff
@@ -0,0 +1,7 @@
+.specification__heading {
+    margin: 10px 10px;
+}
+
+.specification__table {
+    padding: 15px 10px;
+}
```

Comment:
can you change the pixel values to multiples of 8. Chakra uses multiples of 8 for spacing. so just wanna be consistent